packages/docs/SUMMARY.md

```diff
@@ -439,12 +439,14 @@
 - [Election](command-line-interface/election.md)
 - [Exchange](command-line-interface/exchange.md)
 - [Governance](command-line-interface/governance.md)
-- [Locked Gold](command-line-interface/lockedgold.md)
+- [LockedGold](command-line-interface/lockedgold.md)
 - [Network](command-line-interface/network.md)
+- [Oracle](command-line-interface/oracle.md)
+- [ReleaseGold](command-line-interface/releasegold.md)
 - [Node](command-line-interface/node.md)
 - [Transfer](command-line-interface/transfer.md)
-- [Validator Node](command-line-interface/validator.md)
-- [Validator Groups](command-line-interface/validatorgroup.md)
+- [Validator](command-line-interface/validator.md)
+- [ValidatorGroup](command-line-interface/validatorgroup.md)
```

Comment:
I don't quite see why removing the space in the heading is helpful. It doesn't have a space in the command like, but that's just because its in the terminal. I personally think its' more readable and clear with the space here.

Reply:
don't feel it's a big deal, but on balance:
* the section corresponds to a module name
* if I'm using the search, I wouldn't have added a space
* these also often map to contract names which don't have a space

Reply:
hmm but then again the headings of the auto generated pages have spaces... will revert

Reply:
but in fact the auto generated pages use this summary title it appears, so I'll retain this change